cypress/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "types": [
       "cypress",
       "cypress-real-events",
-      "cypress-axe"
+      "cypress-axe",
```

Comment:
Why do we have a `tsconfig.json` file in the Cypress folder? We don't have any tests written in TypeScript - This whole file can be removed

Reply:
it is just a preparation for migration :)

Reply:
Even if we convert Cypress tests to TypeScript (which I don't think we should unless we teach all QAs TypeScript on top of JavaScript), it will not be for a long time so we don't need this file yet. 

YAGNI principle 😉  https://betterprogramming.pub/yagni-you-aint-gonna-need-it-f9a178cd8e1

Reply:
Having said that we might actually be forced to write the tests in TypeScript with this approach which isn't ideal 🤔